apps/foundation-demo/src/app/checkout/effects/checkout.effects.ts

```diff
@@ -22,4 +25,13 @@ export class CheckoutEffects {
       return new ShowReviewView();
     })
   );
+
+  @Effect()
+  onPlaceOrder$ : Observable<any> = this.actions$.pipe(
+    ofType(CheckoutActionTypes.PlaceOrderAction),
+    map(() => {
+      this.router.navigateByUrl('/checkout/thank-you');
```

Comment:
We should wait for a successful response `PlaceOrderSuccess`, right?

Reply:
I added a PlaceOrderSuccess action. PlaceOrder action doesn't do anything yet